src/components/EmojiBlip.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { memo } from 'react'
+import Blip from '@assets/images/blip.svg'
+import Emoji from 'react-native-emoji'
+import { sample } from 'lodash'
+import Box from './Box'
+
+const emojis = {
+  morning: ['coffee', 'fried_egg'],
+  lunch: ['burrito', 'taco', 'pizza'],
+  evening: ['sleeping', 'night_with_stars', 'waning_gibbous_moon'],
+  anytime: [
+    'stuck_out_tongue_winking_eye',
+    'rocket',
+    'balloon',
+    '100',
+    'grin',
+    'the_horns',
+    'star2',
+    'tada',
+    'rainbow',
+  ],
+}
+
+const EmojiBlip = ({ date }: { date: Date }) => {
+  const emojiOptions = [...emojis.anytime]
+  const hour = date.getHours()
+
+  if (hour >= 4 && hour < 12) {
+    emojiOptions.push(...emojis.morning)
+  }
+  if (hour >= 12 && hour < 15) {
+    emojiOptions.push(...emojis.lunch)
+  }
+  if (hour >= 18 || hour < 4) {
```

Comment:
I did `hours >= 17 || hours < 4` in the `TimeOfDayTitle` maybe you can change that to 18 or change this to 17 so they line up?

Reply:
yeah I had that but then most of my afternoon emojis ended up being food related so wanted it closer to lunch time 🙃

Reply:
Ok haha, sounds good then